test/iidm/NodeBreakerVoltageLevelTest.cpp

```diff
@@ -177,6 +180,109 @@ Network createCalculatedBusSwitchTestNetwork() {
     return network;
 }
 
+void addInternalConnection( VoltageLevel::NodeBreakerView& topo, int node1, int node2) {
+    topo.newInternalConnection()
+        .setNode1(node1)
+        .setNode2(node2)
+        .add();
+}
+
+const std::string S5_10K_V = "S5 10kV";
```

Comment:
That's uggly... create a local variable in createNetwork is better!

Reply:
Agree, but this variable is used in test `TestRemoveVoltageLevelWithInternalConnectionsIssue`. 
I updated the test to hardcode the VL name, like it is done in every other test :
```
    network.remove(network.getVoltageLevel("S5 10kV"));
    POWSYBL_ASSERT_THROW(network.getVoltageLevel("S5 10kV"), PowsyblException, "Unable to find to the identifiable 'S5 10kV'");
```